pyflakes/test/test_other.py

```diff
@@ -1262,7 +1265,7 @@ def test_assignInListComprehension(self):
         """
         self.flakes('''
         def f():
-            [None for i in range(10)]
+            return [None for i in range(10)]
```

Comment:
I wonder if list comprehensions might be unassigned when people use them to map a function with side effects to a sequence. For example:

```python
[frobnicate(x) for x in frobnitzen]
```

Personally I use `map()` or a for loop in these instances, but I have seen the above as well.